test/test_helper.rb

```diff
@@ -34,23 +34,27 @@
     "it running? Please see README for setup instructions.")
 end
 
-class Test::Unit::TestCase
-  include Stripe::TestData
-  include Mocha
-
-  setup do
-    Stripe.api_key = "sk_test_123"
-    Stripe.api_base = "http://localhost:#{MOCK_PORT}"
-    stub_connect
-  end
-
-  teardown do
-    Stripe.api_key = nil
-  end
-
-  private
-
-  def stub_connect
-    stub_request(:any, /^#{Stripe.connect_base}/).to_return(body: "{}")
+module Test
+  module Unit
+    class TestCase
```

Comment:
Here we've just unwrapped the module/class into "log-form" and all that changes below is indentation.